waku/waku_core/time.nim

```diff
@@ -33,3 +33,16 @@ template nanosecondTime*(collector: Gauge, body: untyped) =
     metrics.set(collector, nowInUnixFloat() - start)
   else:
     body
+
+# Unused yet
+#[ proc timestampInSeconds*(time: Timestamp): Timestamp =
```

Comment:
If we think (suppose) the value will be an actual time value (not a diff), we don't need to convert it to string. Isn't that cheaper to just compare to some numeric value? Am I miss something? Btw why we expect the value may vary between milli/micro and nano secs?

Reply:
I don't know. @chaitanyaprem wrote this part maybe he can answer.